exchanges/poloniex/poloniex_websocket_trade.go

```diff
@@ -0,0 +1,93 @@
+package poloniex
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/encoding/json"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+)
+
+// WsCreateOrder create an order for an account.
+func (e *Exchange) WsCreateOrder(ctx context.Context, arg *PlaceOrderRequest) (*OrderIDResponse, error) {
+	if err := validateOrderRequest(arg); err != nil {
+		return nil, err
+	}
+	var resp []*OrderIDResponse
+	if err := e.SendWebsocketRequest(ctx, "createOrder", arg, &resp); err != nil {
+		return nil, fmt.Errorf("%w %w", order.ErrPlaceFailed, err)
+	}
+	if len(resp) != 1 {
+		return nil, common.ErrInvalidResponse
```

Comment:
🚧 Change request:
Again: Please use the `order.ErrPlaceFalied` error to wrap this.